### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by cap10morgan**
This is just for rendering (blank for now) vote-by-mail application forms as called for by [this card](https://www.pivotaltracker.com/story/show/109641126).

~~One question for you to weigh in on: This component inherited a lot of Datomic boilerplate from the lein template, but it doesn't *yet* use Datomic. It may someday (to keep track of VBM application statuses, for example), though.~~

~~Should we rip out all the Datomic stuff? Some of it? None of it?~~

UPDATE: We ripped out Datomic.

### CODE REVIEW

**Comment by cap10morgan** [comment]
I owe you README TODO's done, so please ignore this until those are pushed.

**Comment by cap10morgan** [comment]
@tie-rack This should be ready for review now.

File: src/vote_by_mail_works/vbm_form.clj [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+(defn application->vbm-appl-form
+  "Just requests the appropriate template and generates an empty copy of it for
+  now. Eventually it should convert application data structures into the
+  appropriate form-works data structure to render the required VBM application
+  form."
+  [application]
+  (let [voter (:voter application)
+        form (-> application :form name)]
+    {:template (application->template application)
+     :name (str (:first-name voter) "_" (:last-name voter) "_" form ".pdf")
```
Comment (tie-rack) [inline src/vote_by_mail_works/vbm_form.clj]: Should we be concerned about name collisions here?

Comment (cap10morgan) [inline src/vote_by_mail_works/vbm_form.clj]: No, form-works puts them under UUID-named directories.

**Comment by tie-rack** [comment]
LGTM

**Pull request merged**